src/sentry/static/sentry/app/views/performance/transactionSummary/content.tsx

```diff
@@ -152,11 +155,17 @@ class SummaryContent extends React.Component<Props, State> {
     });
   };
 
+  onChangeFilter = (newFilter: SpanOperationBreakdownFilter) => {
+    this.setState({
+      spanOperationBreakdownFilter: newFilter,
+    });
+  };
```

Comment:
Should this push the selection to the URL so when sharing links it persists?

Reply:
I plan to do this in a follow up PR. I had a separate JIRA ticket for this.